modules/google-pubsub/src/parse-pubsub-message.pipe.test.ts

```diff
@@ -0,0 +1,60 @@
+import {PubsubService} from './pubsub.service';
+import {PushMessage} from './domain';
+import {ParsePubsubMessagePipe} from './parse-pubsub-message.pipe';
+import {BadRequestException, HttpException} from '@nestjs/common';
+
+describe('ParsePubsubMessagePipe', () => {
+    let mockPubsubService: PubsubService;
+    let parsePubsubMessagePipe: ParsePubsubMessagePipe<string>;
+
+    beforeEach(() => {
+        mockPubsubService = <any> {
+            decryptMessage: jest.fn().mockReturnValue('{}'),
+            verifyMessage: jest.fn().mockReturnValue(true),
```

Comment:
it is not a good idea to use default values as than you need to overwrite it with `mockReset`, please specify expectation in each test case